specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/securityStandards.json

```diff
@@ -0,0 +1,394 @@
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v2/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete a security standard over subscription scope": {
+            "$ref": "./examples/SecurityStandards/DeleteBySubscriptionSecurityStandard_example.json"
+          },
+          "Delete a security standard over security connector scope": {
+            "$ref": "./examples/SecurityStandards/DeleteBySecurityConnectorSecurityStandard_example.json"
+          },
+          "Delete a security standard over management group scope": {
+            "$ref": "./examples/SecurityStandards/DeleteByManagementGroupSecurityStandard_example.json"
+          }
+        },
+        "tags": [
+          "SecurityStandards"
+        ],
+        "description": "Delete a security standard over a given scope",
+        "operationId": "SecurityStandards_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/StandardId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - security standard was deleted"
+          },
+          "204": {
+            "description": "No Content - security standard does not exist"
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SecurityStandardList": {
+      "type": "object",
+      "description": "Page of a Standard list",
+      "properties": {
+        "value": {
+          "description": "Collection of standards in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityStandard"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "SecurityStandard": {
+      "type": "object",
+      "description": "Security Standard on a resource",
+      "properties": {
+        "properties": {
+          "description": "Properties of a security standard",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/StandardProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
```

Comment:
>"../../../common/v1/types.json#/definitions/Resource" [](http://example.com/codeflow?start=18&length=53)

consider using "../../../../../common-types/resource-management/v5/types.json#/definitions/TrackedResource" which include "systemData":[https://github.com/Azure/azure-openapi-validator/blob/main/docs/required-read-only-system-data.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/required-read-only-system-data.md)

Reply:
It's not a tracked resource